ivy_tests/test_ivy/conftest.py

```diff
@@ -50,6 +83,81 @@ def pytest_configure(config):
         backend_strs = available_frameworks
     else:
         backend_strs = raw_value.split(",")
+    no_mp = config.getoption("--no-mp")
+
+    if not no_mp:
+        # we go multiprocessing, if  multiversion
+        known_backends = {"tensorflow", "torch", "jax"}
+        found_backends = set()
+        for fw in backend_strs:
+            if "/" in fw:
+                multiprocessing_flag = True
+                # spin up multiprocessing
+                # build mp process, queue, initiation etc
+                input_queue = mp.Queue()
+                output_queue = mp.Queue()
+                proc = mp.Process(target=backend_proc, args=(input_queue, output_queue))
+                # start the process so that it loads the framework
+                input_queue.put(fw)
+                proc.start()
+
+                # we have the process running, the framework imported within,
+                # we now pack the queue and the process and store it in dict
+                # for future access
+                mod_backend[fw.split("/")[0]] = (proc, input_queue, output_queue)
+                # set the latest version for the rest of the test code and move on
+                default_framework_mapper(fw.split("/")[0], set_too=True)
+                found_backends.add(fw.split("/")[0])
```

Comment:
No need to do the splitting more than once, let's do it once and save the value for it, makes it clear to know what value is it. `truncated_fw=fw.split("/")[0]` for example.

Reply:
Noted